ntbs-service/wwwroot/source/Components/ValidateMultiple.ts

```diff
@@ -66,6 +84,23 @@ function buildKeyValuePairsQueryString(keys: Array<string>, values: Array<string
     return qs.stringify(queryStringObject);
 };
 
+function buildKeyDateValuesQueryString(keys: Array<string>, days: Array<string>, months: Array<string>, years: Array<string>) {
+    const queryStringObject: any = { keyValuePairs: [] };
+    for (let i = 0; i < keys.length; i++) {
+        queryStringObject["keyValuePairs"].push({ key: keys[i], day: days[i], month: months[i], year: years[i] });
+    }
+    return qs.stringify(queryStringObject);
+};
+
+function arrayContainsEmptyValues(array: Array<string>) {
+    for (let i = 0; i < array.length; i++) {
+        if (!array[i]) {
+            return true;
+        }
+    }
+    return false;
+}

```

Comment:
Instead of this we should be able to just use a `array.some(el => !el)` I think?

Reply:
Nice, I was hoping there would be a much neater way haha